Do not use `*` in `SELECT` queries for Oracle (#917)
tidyverse/dbplyr: NEWS.md

# dbplyr (development version)

> * Querying Oracle databases works again. Unfortunately, the fix caused to
>   explicitly select every column again for Oracle databases (@mgirlich, #908).

Reviewer: ```suggestion
* Querying Oracle databases works again. Unfortunately, the fix requires every
  column to be explicitly selected again (@mgirlich, #908).
```
